app/views/service_manual/index.html.erb

```diff
@@ -1,47 +1,55 @@
-<% content_for :title, "Service Manual" %>
-
-<header class="app-hero">
-  <div class="app-hero__content govuk-width-container">
-    <div class="govuk-main-wrapper">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-          <%= render "govuk_publishing_components/components/heading", {
-            text: "Service Manual",
-            font_size: "xl",
-            heading_level: 1,
-            inverse: true,
-            margin_bottom: 6,
+<%
+  content_for :title, "Service Manual"
```

Comment:
I think this is supposed to be "Service Manual - GOV.UK"

In other formats it's been rendered as `<% content_for :title, "#{content_item.title} - GOV.UK" %>`

Reply:
Thanks, have updated in a separate commit.